client/src/js/components/bhDepotSelect/bhDepotSelect.tmpl.html

```diff
@@ -1,6 +1,6 @@
 <div ng-form="DepotForm" bh-depot-select ng-model-options="{ updateOn: 'default' }">
   <div
-    class="form-group"
+    class="form-group has-feedback"
```

Comment:
Is this necessary?  it seems like we have some forms that use the $loading flag that do not use it.

Reply:
Yes, to show the icon on the input, the `has-feedback` class does the magic.